export.config.json

```diff
@@ -6,6 +6,7 @@
     "Input": "input",
     "Form": "form",
     "Icons": "icons",
-    "Core": "core"
+    "Core": "core",
+    "TextLink": "textlink"
```

Comment:
the last element of a json should contain a `,` for git diff

Reply:
I agree, but its not feasible to add it everytime manually. I'll be making improvements to our husky/eslint setup to take care of that